FitGoal/FitGoal/HomeViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ViewController.swift
+//  FitGoal
+//
+//  Created by Reynaldo Aguilar on 2/3/20.
+//  Copyright © 2020 Eliany Barrera. All rights reserved.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController {
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        configureBackgroundView()
```

Comment:
Why do you configure the view every time the view appears? Remember, if you want to do a configuration only once you should use the `viewDidLoadInstead`.